funnel/forms/helpers.py

```diff
@@ -227,6 +227,14 @@ def image_url_validator():
     )
 
 
+def video_url_list_validator(form, field):
+    for url in field.data:
+        try:
+            parse_video_url(url)
+        except ValueError as exc:
+            raise forms.validators.StopValidation(str(exc))
```

Comment:
Don't expose error strings to UI. No i18n, and no insight into content. Have a local message here.